native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -67,11 +68,10 @@ const styles = StyleSheet.create({
     flexGrow: 1,
     display: 'flex',
     flexDirection: 'row',
-    alignItems: 'center',
     paddingBottom: spacingSm,
     paddingLeft: spacingMd,
     paddingRight: spacingMd,
-    paddingTop: spacingSm,
+    paddingTop: ((spacingXl - (2 * borderSizeSm)) - spacingBase) / 2,
```

Comment:
this obvs works, but I'm particularly a fan on this kind of arbitrary operations to achieve a result, is there a thinking behind it?

Reply:
The thinking behind it was:
the whole thing is spacingXl tall, but we need to account for the borders at the top and bottom,
So the space we have to align to is `spacingXl - (2 * borderSizeSm)`
Then, the padding is half the remaining space after we take away the icon which has height `spacingBase`

I appreciate it's not pretty, but not sure how else to achieve this!

Reply:
I totally get that, that's why I've asked about the thinking, if I can think of a better way I'll update this

Reply:
Ok :)